overturemaps/core.py

```diff
@@ -46,6 +93,10 @@ def record_batch_reader(
         release = ALL_RELEASES[-1]
     path = _dataset_path(overture_type, release)
 
+    intersecting_files = None
+    if bbox and not no_stac:
```

Comment:
this is a good example that reads better if you invert and make it `if bbox and stac`.